travis/install_pg

```diff
@@ -4,6 +4,13 @@
 
 set -eux
 
+# If USE_CUSTOM_PG_BUILD variable is set, we will compile PostgreSQL from the source.
+if [ "${USE_CUSTOM_PG_BUILD-0}" != "0" ]; then
```

Comment:
For brevity, can you (a) abbreviate to `USE_CUSTOM_PG` and (b) just use the convention that if it's not defined or empty, it's false. Like this:

```bash
if [ -z "${USE_CUSTOM_PG} ]; then
  # ...
```